src/commands/trialApp/showTutorial.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { commands, extensions, Uri } from 'vscode';
+import { AzExtTreeItem, IActionContext } from 'vscode-azureextensionui';
+import { TrialAppTreeItem } from '../../explorer/trialApp/TrialAppTreeItem';
+import { delay } from '../../utils/delay';
+import { ext } from './../../extensionVariables';
+
+export async function showTutorial(context: IActionContext, node?: TrialAppTreeItem): Promise<void> {
+
+    const timeoutInSeconds: number = 60;
+    const maxTime: number = Date.now() + timeoutInSeconds * 1000;
+    const extensionId: string = 'redhat.vscode-didact';
+
+    if (!node) {
+        const children: AzExtTreeItem[] = await ext.azureAccountTreeItem.getCachedChildren(context);
+        children.forEach((child: AzExtTreeItem) => {
+            if (child instanceof TrialAppTreeItem) {
+                node = child;
+            }
+        });
+    }
+
+    if (extensions.getExtension(extensionId)) {
+        showDidact();
+    } else {
+        const commandToRun: string = 'extension.open';
+        commands.executeCommand(commandToRun, extensionId);
+
+        while (Date.now() < maxTime) {
```

Comment:
> I'm not really an expert on the tutorial stuff myself either, so I'm hoping @fiveisprime will spruce it up with some PM magic for us

Yeah I have some changes and polished to make with the tutorial. But I wanted to get this PR out Asap in case there were any code concerns.